artemis-protocols/artemis-stomp-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/stomp/StompFrame.java

```diff
@@ -103,7 +103,7 @@ public ActiveMQBuffer toActiveMQBuffer() throws Exception {
             return buffer;
          }
 
-         StringBuilder head = new StringBuilder();
+         StringBuilder head = new StringBuilder(512);
```

Comment:
did you run any benchmarks to find this? 

I agree .. just wondering how bad was it.